recipes/bamstats/meta.yaml

```
@@ -0,0 +1,28 @@
+{% set name = "bamstats" %}
+{% set version = "0.3.5" %}
+
+package:
+  name: {{name}}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  - url: https://github.com/guigolab/{{name}}/archive/v{{ version }}.tar.gz
+    sha256: 0b397c0b09c4a944e564dd9e92eb5e9e8408e97cf1a54af07dced8783342b977
+    folder: src/github.com/guigolab/{{name}}/
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+
+test:
+  commands:
+    - {{name}} --help
```

```suggestion
    - bamstats --help
```
I'd suggest using the actual binary here to make things more explicit.